# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_bash.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./locale.c~	Tue Jul 30 19:26:16 2002
+++ ./locale.c	Tue Jul 30 19:25:47 2002
@@ -190,7 +190,11 @@
 set_lang (var, value)
      char *var, *value;
 {
-  return ((lc_all == 0 || *lc_all == 0) ? setlocale (LC_ALL, value?value:"") != NULL : 0);
+#if 0
+	return ((lc_all == 0 || *lc_all == 0) ? setlocale (LC_ALL, value?value:"") != NULL : 0);
+#else
+	return 0;
+#endif
 }
 
 /* Get the value of one of the locale variables (LC_MESSAGES, LC_CTYPE) */
--- ./lib/sh/oslib.c.orig	Sat May 11 18:10:48 2002
+++ ./lib/sh/oslib.c	Sat May 11 18:11:17 2002
@@ -214,7 +214,7 @@
 #  endif /* !HAVE_UNAME */
 #endif /* !HAVE_GETHOSTNAME */
 
-#if !defined (HAVE_KILLPG)
+#if 0
 int
 killpg (pgrp, sig)
      pid_t pgrp;
--- ./support/mksignames.c.orig	2002-10-09 12:19:12.000000000 +0200
+++ ./support/mksignames.c	2002-10-09 12:20:58.000000000 +0200
@@ -426,5 +426,7 @@
 
   initialize_signames ();
   write_signames (stream);
+
+  if (argc == 2) fclose(stream);
   exit (0);
 }
Binary files ./mksignames.orig and ./mksignames differ
